testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -1,34 +1,37 @@
 import pytest
 import random
 import time
-import os
+from collections import OrderedDict
 
+from couchbase.bucket import Bucket
+from couchbase.n1ql import N1QLQuery
 from CBLClient.Database import Database
-from keywords.utils import log_info
+from CBLClient.Authenticator import Authenticator
+from CBLClient.MemoryPointer import MemoryPointer
 from CBLClient.Replication import Replication
 from keywords.MobileRestClient import MobileRestClient
-from CBLClient.Authenticator import Authenticator
-from libraries.testkit.cluster import Cluster
-from couchbase.n1ql import N1QLQuery
-from couchbase.bucket import Bucket
 from keywords.constants import SDK_TIMEOUT
-from CBLClient.MemoryPointer import MemoryPointer
 from keywords.couchbaseserver import CouchbaseServer
-from CBLClient.Query import Query
+from keywords.utils import log_info
+from libraries.testkit.cluster import Cluster
+from testsuites.CBLTester.CBL_Functional_tests.SuiteSetup_FunctionalTests.test_query import test_get_doc_ids, \
+    test_any_operator, test_doc_get, test_get_docs_with_limit_offset, test_multiple_selects, test_query_where_and_or, \
+    test_query_pattern_like, test_query_pattern_regex, test_query_isNullOrMissing, test_query_ordering, \
+    test_query_substring, test_query_collation, test_query_join, test_query_inner_join, test_query_cross_join, \
+    test_query_left_join, test_query_left_outer_join, test_equal_to, test_not_equal_to, test_greater_than, \
+    test_greater_than_or_equal_to, test_less_than, test_less_than_or_equal_to, test_in, test_between, test_is, \
+    test_isnot, test_not, test_single_property_fts, test_multiple_property_fts, test_fts_with_ranking, \
+    test_getDoc_withValueTypeDouble, test_getDoc_withLocale, test_query_arthimetic
```

Comment:
looks like when new query test added, we should add it here, right?

Reply:
Yes, we have to add the new query test in the list. I couldn't find an easier way, so I had to hard code it